modules/blog/blog-posts-list/blog-posts-list.module.scss

```diff
@@ -0,0 +1,142 @@
+@import '../../../styles/variables';
+@import '../../../styles/mixins';
+
+.container {
+  display: grid;
+  column-gap: 1.5rem;
+  row-gap: 0.75rem;
+  grid-template-columns: 24rem 38rem 11rem;
+  grid-template-rows: 1.25rem 5.625rem;
+  grid-template-areas:
+    'category category category'
+    'title description authorWrapper';
+  position: relative;
+
+  padding: 1.5rem 1.35rem 3rem;
+
+  @include breakpoint(medium-only) {
+    display: flex;
+    flex-direction: column;
+    padding: 1.5rem 1.35rem 3rem;
+  }
+
+  @include breakpoint(small) {
+    display: flex;
+    flex-direction: column;
+    padding: 1.5rem 0;
+  }
+}
+
+.imageWrapper {
+  left: 4rem;
+  opacity: 0.5;
+  position: absolute;
+  top: -1.2rem;
+  visibility: hidden;
+  z-index: -1;
+
+  @include breakpoint(small) {
+    top: -0.2rem;
+  }
+}
+
+.imageWrapperShown {
+  visibility: visible;
+}
```

Comment:
It would be nice if we could fade this in. Right now it feels a little too quick.

Reply:
i added this now 😄